docs/manual/faq.rst

```diff
@@ -75,4 +75,52 @@ LibreOffice menus don't appear or don't stay visible
 A workaround for problem with the mouse in libreoffice is setting the environment variable »SAL_USE_VCLPLUGIN=gen«.
 It is dependet on your system configuration where to do this. e.g. ArchLinux with libreoffice-fresh in /etc/profile.d/libreoffice-fresh.sh.
 
+How can I get my groups to stick to screens?
+============================================
 
+This behaviour can be replicated by configuring your keybindings to not move
+groups between screens. For example if you want groups ``"1"``, ``"2"`` and
+``"3"`` on one screen and ``"q"``, ``"w"``, and ``"e"`` on the other, instead
+of binding keys to ``lazy.group[name].toscreen()``, use this:
+
+.. code-block:: python
+
+    def go_to_group(name: str) -> Callable:
+        def _inner(qtile: Qtile) -> None:
+            if len(qtile.screens) == 1:
+                qtile.groups_map[name].cmd_toscreen()
+                return
+
+            if name in '123':
+                qtile.focus_screen(0)
+                qtile.groups_map[name].cmd_toscreen()
+            else:
+                qtile.focus_screen(1)
+                qtile.groups_map[name].cmd_toscreen()
+
+        return _inner
+
+    for i in groups:
+        keys.append(Key([mod], i.name, lazy.function(go_to_group(i.name))))
```

Comment:
Calling `go_to_group` here creates the `_inner` function set up with that group's name, so rather than a function with a parameter, it's a closure. I realise now though it could just be a function with a parameter. I wonder why I did it this way... Maybe the parameter way is preferable for new users as it's less confusing Python?

Reply:
Ah. Indeed. I think it's ok as is then.